src/WebJobs.Extensions.OpenAI/Agents/ChatBotService.cs

```diff
@@ -82,11 +82,11 @@ void Initialize(ChatBotCreateRequest request)
             request.Id,
             request.Instructions ?? "(none)");
 
-        this.InitialState = new ChatBotRuntimeState
+        this.initialState = new ChatBotRuntimeState
         {
             ChatMessages = string.IsNullOrEmpty(request.Instructions) ?
-                new List<ChatMessageEntity>() :
-                new List<ChatMessageEntity>() { new ChatMessageEntity(request.Instructions, ChatRole.System.ToString()) },
+                new() :
+                new() { new ChatMessageEntity(request.Instructions, ChatRole.System.ToString()) },
```

Comment:
I should have been more clear. The point is not to remove the class name in all instances. Rather, the point is to remove redundant cases. The problem with this particular change is that it's not obvious what `new()` corresponds to. The previous version is actually preferred.
```suggestion
                new List<ChatMessageEntity>() { new ChatMessageEntity(request.Instructions, ChatRole.System.ToString()) },
```